config/routes.rb

```diff
@@ -18,11 +18,11 @@
     get "/test-search/search/opensearch" => "search#opensearch"
   end
 
-  # Q&A checklist frontend for brexit guidance
-  get '/get-ready-brexit-check/results' => 'checklist#results', as: :checklist_results
-  get '/get-ready-brexit-check/questions' => 'checklist#show', as: :checklist_questions
-  get '/get-ready-brexit-check/email-signup' => 'checklist#email_signup', as: :checklist_email_signup
-  post '/get-ready-brexit-check/email-signup' => 'checklist#confirm_email_signup', as: :checklist_confirm_email_signup
+  # Q&A brexit_checker frontend for brexit guidance
```

Comment:
```suggestion
  # Routes for the for Brexit Checker
```
Might be a less confusing comment?

Reply:
Sure: https://github.com/alphagov/finder-frontend/pull/1535